src/main/java/algorithms/curated170/medium/numberofconnectedcomponents/NumberOfConnectedComponentsDFS.java

```diff
@@ -0,0 +1,62 @@
+package algorithms.curated170.medium.numberofconnectedcomponents;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NumberOfConnectedComponentsDFS {
+
+    boolean[] visited;
+    List<List<Integer>> connections = new ArrayList<>();
+
+    public int countComponents(int[][] edges, int n) {
```

Comment:
Very nice solution.
Can we alter the params order:
   public int countComponents(int n, int[][] edges) 
   
   
![image](https://user-images.githubusercontent.com/60903744/118022003-88e8ed00-b364-11eb-8c02-6e214be19580.png)